packages/sdk/src/send/send-get-events.js

```diff
@@ -53,3 +64,28 @@ export async function sendGetEvents(ix, opts = {}) {
 
   return ret
 }
+
+export async function sendGetEvents(ix, opts = {}) {  
+  ix = await ix
+ 
+  if (isHeightRangeRequest(ix)) {
+    return getEventsForHeightRange(
+      ix.events.eventType,
+      Number(ix.events.start),
+      Number(ix.events.end),
+      opts,
+      ix.tag,
+    )
+  }
+
+  if (isBlockIdsRequest(ix)) {
+    return getEventsForBlockIDs(
+      ix.events.eventType,
+      ix.events.blockIds,
+      opts,
+      ix.tag,
+    )
+  }
+
+  throw new Error("Invalid event query, must specify height range or block IDs")
+}
```

Comment:
Can you please add a changelog entry for this to the SDKs changelog file => https://github.com/onflow/flow-js-sdk/blob/master/packages/sdk/CHANGELOG.md